c2corg_ui/static/partials/cards/articles.html

```diff
@@ -0,0 +1,20 @@
+<a ng-href="{{cardCtrl.createURL()}}" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+  <div class="outing-author text-center">
```

Comment:
Not sure to get what you mean.
But anyway we don't need such data in the article cards so this ``<div class="outing-author text-center">`` and its content can be removed => problem fixed, isn't it?